android/src/main/java/com/rnmaps/maps/MapView.java

```diff
@@ -523,25 +523,30 @@ public void setBoundary(ReadableMap boundary) {
 
   // called as soon as the map is ready. Assuming nothing was set on the map at this point.
   private void applyBridgedProps() {
-    if(boundary != null) {
-      map.setLatLngBoundsForCameraTarget(boundary);
-    }
-    if(initialRegion != null) {
-      moveToRegion(initialRegion);
-      initialRegionSet = true;
-    } else if(initialCamera != null) {
-      moveToCamera(initialCamera);
-      initialCameraSet = true;
-    } else if(region != null) {
-      moveToRegion(region);
-    } else {
-      moveToCamera(camera);
-    }
-    if(customMapStyleString != null) {
-      map.setMapStyle(new MapStyleOptions(customMapStyleString));
+    try {
+      if(boundary != null) {
+        map.setLatLngBoundsForCameraTarget(boundary);
+      }
+      if(initialRegion != null) {
+        moveToRegion(initialRegion);
+        initialRegionSet = true;
+      } else if(initialCamera != null) {
+        moveToCamera(initialCamera);
+        initialCameraSet = true;
+      } else if(region != null) {
+        moveToRegion(region);
+      } else {
+        moveToCamera(camera);
+      }
+      if(customMapStyleString != null) {
+        map.setMapStyle(new MapStyleOptions(customMapStyleString));
+      }
+    } catch (Exception e) {
+        System.err.println("Fail during applying MapView props");
```

Comment:
@mateki0 I would say never silently catch an exception in this library, especially when it's a configuration issue by the user of the library.

and please avoid moving functionality from native to react-native because the same native code might end up being called from a different place from react-native and the behaviour becomes unpredictable